src/voodoo/spellbook/spells/trueStrike.ts

```diff
@@ -27,7 +27,7 @@ export const trueStrike: SpellFunction = async (voodoo, accountId, upgradeConfig
     }
   });
 
-  const multiplier = 1 + attributes.intensify / 100;
+  const multiplier = attributes.intensify / 100;
```

Comment:
Implementing mdigena's suggestions
```suggestion
  const bonus = attributes.intensify / 100;
```